docs/deployment.md

```diff
@@ -0,0 +1,67 @@
+If you provisioned your servers on OpenStack with the example playbook you must run this.
+
+### [`idr-01-install-idr.yml`](../ansible/idr-01-install-idr.yml)
+This is the main playbook that does almost all the work involved in setting up the IDR, including installing and setting up PostgreSQL, OMERO.server, OMERO.web, and the front-end caching Nginx proxy.
+
+### [`idr-03-postinstall.yml`](../ansible/idr-03-postinstall.yml)
+This is an optional playbook to set up some OMERO users, including setting the OMERO `root` user password to a random string (stored in `/root/idr-root-ice.config`) and creating a public user for OMERO.web.
+This playbook is *not* idempotent.
+
+### [`idr-09-monitoring.yml`](../ansible/idr-09-monitoring.yml)
+This is an optional playbook to set up monitoring of the IDR and OMERO.server.
+This includes configuring all servers with [Munin](http://munin-monitoring.org/), and enabling Slack notifications for OMERO.server errors.
+You will need to provide a secret Slack token for Slack notifications.
+
+
+## Ansible inventory
+
+You have two options for setting up an Ansible inventory to use with the IDR deployment playbooks, depending on how you provisioned the servers.
+
+### OpenStack dynamic private inventory
+This is the recommended method if you provisioned the servers using the supplied OpenStack playbook and are familiar with [Ansible dynamic inventories](http://docs.ansible.com/ansible/intro_dynamic_inventory.html), and is the method we use internally for managing the production IDR servers.
+
+[inventories/openstack-idr.py](../inventories/openstack-idr.py) is a modified version of the [default Ansible OpenStack dynamic inventory](https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py) to make it easier to use an [Ansible SSH gateway/jump host](http://docs.ansible.com/ansible/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to), and to configure multiple networks.
+The main changes are:
+- automatic setting of SSH ProxyCommand
+- ordering of network interfaces
+
+This inventory will attempt to automatically route all SSH access via `idr-proxy`, using metadata attached to the servers in the [IDR openstack-idr-instance](https://github.com/IDR/ansible-role-openstack-idr-instance) role.
+
+### Static inventory
+This is recommended if you have provisioned the servers yourself, or if you are unfamiliar with setting up an Ansible dynamic inventory.
+Modify the `Hosts` section in the [example static inventory](../inventories/ansible-hosts).
+You should not need to change the `Host groups` section.
+
+## Ansible deployment example
+
+You must run Ansible from the [`ansible`](../ansible) directory.
+
+If you are using a static inventory run:
+
+    ansible-playbook -i ../inventories/ansible-hosts --diff -e idr_environment=idr idr-01-install-idr.yml idr-03-postinstall.yml
```

Comment:
What does that mean? Preferable would be if users didn't have to add `-e idr_environment=idr` for commands if it's not needed.